frame/membership/src/lib.rs

```diff
@@ -107,34 +152,28 @@ decl_event!(
 		/// One of the members' keys changed.
 		KeyChanged,
 		/// Phantom member, never used.
-		Dummy(sp_std::marker::PhantomData<(AccountId, Event)>),
+		Dummy(PhantomData<(T::AccountId, <T as Config<I>>::Event)>),
 	}
-);
 
-decl_error! {
-	/// Error for the nicks module.
-	pub enum Error for Module<T: Config<I>, I: Instance> {
+	/// Old name generated by `decl_event`.
+	#[deprecated(note = "use `Event` instead")]
+	pub type RawEvent<T, I = ()> = Event<T, I>;
```

Comment:
We should just remove this and make it so that all importers of this pallet use `Event` instead of `RawEvent`.

Reply:
I think it is enough to specify it as deprecated. After all pallets are updated to the new macro syntax and the major version of the crate is upgraded, then this part should be considered to be deleted.